lesson_8/index8.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <link href="styless8.css" rel="stylesheet" type="text/css">
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document HW</title>
+</head>
+<body>
+    <div>
+        <h1 class="line">Iryna Masiuk</h1>
+        <form action="C:/всяке/Robot_Dreams/lesson_6/index6.html">
+            <input type="submit" value="Offer job" name="offer_job">
+        </form>
+        <br>
+        <img src="myphoto.jpg" width="240" height="320">
+        <h3 class="line">Junior frontend developer</h3>
+    </div>
+    <p class="line"><strong><u>Work Experince</u></strong></p>
+    <table border="2">
+        <tr>
+            <th>Period</th>
+            <th>Position</th>
+        </tr>
+        <tr>
+            <td>2017 - 2018</td>
+            <td>Consultant of the contact center of <a href="https://www.oschadbank.ua/" target="_blank">Oschad Bank</td>
+        </tr>
+        <tr>
+            <td>2018 - 2019</td>
+            <td>Accountant at <a href="https://firm.ua/ua/kyiv/" target="_blank">Firm.ua</a></td>
+        </tr>
+        <tr>
+            <td>2019 - 2021</td>
+            <td>Database analyst at <a href="https://blago.ua/" target="_blank">Blago</a></td>
+        </tr>
+        <tr>
+            <td>2021 - 2022</td>
+            <td>Technical support consultant <a href="https://blago.ua/" target="_blank">Blago</a></td>
+        </tr>
+        <tr>
+            <td>2022 - 2023</td>
+            <td>Business analyst at <a href="https://www.creatio.com/" target="_blank">Creatio</a></td>
+        </tr>
+    </table>
+    <div>
+        <p class="line"><strong><u>Skills:</u></strong></p>
```

Comment:
tags strong and u already are not needed you know how to make the same by css styles

Reply:
I did